user-guide/Advanced_Modules/DOM/Tutorials/DOM_Remove_Enum_Entry.md

```diff
@@ -0,0 +1,406 @@
+
+            // Create the domDefinition
+            var personDomDefinition = new DomDefinition()
+            {
+                Name = "PersonDomDefinition",
+                SectionDefinitionLinks = new List<SectionDefinitionLink>()
+                {
+                    new SectionDefinitionLink(sectionDefinition?.GetID())
+                }
+            };
+
+            personDomDefinition = _domHelper.DomDefinitions.Create(personDomDefinition);
+
+            var sectionDefinitionId = sectionDefinition?.ID;
+            var firstNameFieldDescriptorId = firstNameFieldDescriptor.ID;
+            var lastNameFieldDescriptorId = lastNameFieldDescriptor.ID;
+            var phoneNumberFieldDescriptorId = phoneNumberFieldDescriptor.ID;
+            var jobTitleFieldDescriptorId = jobTitleFieldDescriptor.ID;
+            var domDefinitionId = personDomDefinition.ID;
+
+            // Create a list of FieldValues for the general Section
+            var fieldValues = new List<FieldValue>()
+            {
+                new FieldValue(firstNameFieldDescriptorId, new ValueWrapper<string>("John")),
+                new FieldValue(lastNameFieldDescriptorId, new ValueWrapper<string>("Doe")),
+                new FieldValue(phoneNumberFieldDescriptorId, new ValueWrapper<string>("0423482635")),
+                new FieldValue(jobTitleFieldDescriptorId, new ValueWrapper<int>(0)),
+            };
+
+            // Add them to a Section linked to the ID of the GeneralSectionDefinition
+            var generalSection = new Section()
+            {
+                SectionDefinitionID = sectionDefinitionId
+            };
+
+            foreach (var fieldValue in fieldValues)
+            {
+                generalSection.AddOrReplaceFieldValue(fieldValue);
+            }
+
+            // Create of the actual DomInstance
+            var domInstance = new DomInstance()
+            {
+                DomDefinitionId = domDefinitionId,
+                Sections = { generalSection }
+            };
+            domInstance = _domHelper.DomInstances.Create(domInstance);
+        }
+    }
+}
+```
+
+To visualize the setup, create a low code app. On that low code app:
+1. Drag the DOM definition: *"Object Manager Definitions" > "people_app" > "PersonDomDefinition"*
+1. Choose the "Form" visualization
+1. Create a query that shows all instances: *"Queries" > "Get object manager instances" > "people_app" > "PersonDomDefinition"*
+1. Drag the query and choose "Table" visualization
+
+   ![Step1](~/user-guide/images/DOM_Remove_Enum_Entry_Step1_1.png)
+
+   ![Step1_2](~/user-guide/images/DOM_Remove_Enum_Entry_Step1_2.png)
+
+## Step 2: Soft-delete the enum entry
+
+The given code does the following:
+1. Gets the enum entry from the DOM definition.
+1. Sets the property "IsSoftDeleted" of the enum entry to true.
+1. Updates the DOM definition.
+
+```C#
```

Comment:
Copy-pasting this code block in an automation script gives me 1 error after validating:
"IGenericEnumEntry does not contain a definition for IsSoftDeleted and no accessible extension method 'IsSoftDeleted' accepting a first argument of type 'Bar.placeOrderForm' could be found (are you missing a using directive or an assembly reference?"